element.go

```diff
@@ -0,0 +1,62 @@
+package pokemonbattlelib
+
+import "fmt" 
+type ElementalType uint32
+var effect int
+
+const (
+	Normal ElementalType = 65536 >> iota
```

Comment:
It'll be easier to work with if you do `1 << iota` instead

Reply:
Wouldn't the chart be flipped?